apps/docs/pages/guides/auth/quickstarts/react.mdx

```diff
@@ -78,13 +78,23 @@ export const meta = {
 
       ```js src/index.js
         import './index.css'
+        import { AppState } from 'react-native'
```

Comment:
might be a bit confusing to mix native and browser since this isn't a native-only tutorial

what do you think about adding a step below this for the changes here, so we can explicitly mark it as native-only?

Reply:
I'm guessing this edit was meant to be fore this page instead?
https://supabase.com/docs/guides/auth/quickstarts/react-native